src/unittests/f32_interval.spec.ts

```diff
@@ -3958,14 +3959,512 @@ g.test('subtractionMatrixInterval')
     );
   });
 
-interface MatrixScalarCase {
+g.test('multiplicationMatrixMatrixInterval')
+  .paramsSubcasesOnly<MatrixPairToMatrixCase>([
+    // Only testing that different shapes of matrices are handled correctly
+    // here, to reduce test duplication.
+    // multiplicationMatrixMatrixInterval uses and transposeInterval &
+    // dotInterval for calculating intervals, so the testing for those functions
+    // will cover the actual interval calculations.
+    {
+      input: [
+        [
+          [1, 2],
+          [3, 4],
+        ],
+        [
+          [10, 20],
+          [30, 40],
+        ],
+      ],
+      expected: [
+        [70, 100],
+        [150, 220],
+      ],
+    },
+    {
+      input: [
+        [
+          [1, 2],
+          [3, 4],
+        ],
+        [
+          [10, 20],
+          [30, 40],
+          [50, 60],
+        ],
+      ],
+      expected: [
+        [70, 100],
```

Comment:
Applying the same rules from above, we'd need the vector length of the LHS row to be equal to the vector length of the RHS column, no?

Reply:
Once transposed, this makes sense.